src/Core/src/Core/Extensions/ILayoutExtensions.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Microsoft.Maui
+{
+	public static class ILayoutExtensions
+	{
+		public static void InvalidateChildrenIsEnabled(this ILayout layout)
+		{
+			foreach (var children in layout.OrderByZIndex())
```

Comment:
Even if this were the correct place to do it, there's no reason to enable/disable the children in z-index order.